factcast-store-validation/src/test/java/org/factcast/store/pgsql/internal/PgSynchronizedQueryTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2018 factcast (http://factcast.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.factcast.store.pgsql.internal;
+
+import static org.mockito.ArgumentMatchers.startsWith;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+import org.factcast.store.pgsql.internal.query.PgLatestSerialFetcher;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Answers;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.PreparedStatementSetter;
+import org.springframework.jdbc.core.RowCallbackHandler;
+
+@ExtendWith(MockitoExtension.class)
+public class PgSynchronizedQueryTest {
+
+    PgSynchronizedQuery uut;
+
+    @Mock(answer = Answers.RETURNS_DEEP_STUBS)
+    JdbcTemplate jdbcTemplate;
+
+    String sql = "SELECT 42";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/uwe/factcast/pullRequest?prid=4877539)